README.md

```diff
@@ -9,6 +9,8 @@ This desktop application aims to simplify the process of Attendance Management o
 ## About the Project
 This python based app uses OpenCV libraries: face detection using Haar feature-based cascade classifier and face recognition using Local Binary Patterns Histograms (LBPH) and openpyxl library to manage excel sheet through python scripts. GUI features are implemented using tkinter library.
 
+<img src="logo/logo.png" width="500" height="350">
```

Comment:
⚠️ Incorrect image path. No image is being rendered on README

Reply:
Application: PixelLab
Icon : Raw icon from Shutterstock(modified later on PS)
I have added design files in the projectlogo folder
Even corrected image path

Reply:
https://play.google.com/store/apps/details?id=com.imaginstudio.imagetools.pixellab